src/NHibernate.Test/Criteria/DetachedCriteriaSerializable.cs

```diff
@@ -391,13 +391,16 @@ public void ExecutableCriteria()
 			SerializeAndList<Student>(dc);
 
 			// Subquery
-			dc = DetachedCriteria.For(typeof(Student))
-				.Add(Property.ForName("StudentNumber").Eq(232L))
-				.SetProjection(Property.ForName("Name"));
+			if (TestDialect.SupportsOperatorAll)
```

Comment:
I don't get it. Why did it work for SQLite in non-generic version?

Reply:
Because test was wrong before... Frederic've noticed it and fixed.
It was something like
```
var dc = query;
var dcs = subquery;
SerializeAndList(dc); <- subquery is not used.. so non-generic version works
```

Reply:
Ok